endpoints/openrtb2/auction.go

```diff
@@ -1821,3 +1829,28 @@ func checkIfAppRequest(request []byte) (bool, error) {
 	}
 	return false, nil
 }
+
+func (deps *endpointDeps) setIntegrationType(req *openrtb_ext.RequestWrapper, account *config.Account) error {
+	reqExt, err := req.GetRequestExt()
+	if err != nil {
+		return err
+	}
+	reqPrebid := reqExt.GetPrebid()
+	if reqPrebid == nil {
```

Comment:
Do we want to verify `account` isn't nil here to ensure this method doesn't fail when attempting to access fields on the account? Technically the account shouldn't be nil given the current state of the code because the calling code makes sure the account exists before calling this function but adding this defensive programming check would make this more robust. @mansinahar thoughts?

Reply:
Yeah, I agree with @bsardo. We should add an account not nil check as well before we access the Default Integration